src/pages/cw-storage-plus/basics.mdx

```diff
@@ -0,0 +1,82 @@
+import { Callout } from "nextra/components";
+
+# Basics
+
+## Containers
+
+_cw-storage-plus_ provides a number of storage containers that can be used to
+store data on the blockchain. The fundamental ones are
+
+- [`Item`](containers/item),
```

Comment:
I would maybe remove the commas at the end of the list

Reply:
If that's how everyone's been doing lists, I'm happy to follow suit, sure. In general this isn't wrong AFAIK; it's just a question of "which style guide you're following".